types/foundry-vtt-api/types/apps/forms/forms.d.ts

```diff
@@ -0,0 +1,60 @@
+// @TODO:
+
+/**
+ * Configure the Combat tracker to display additional information as appropriate
+ */
+declare class CombatTrackerConfig extends FormApplication {}
+
+/**
+ * Edit a folder, configuring its name and appearance
+ * @type {FormApplication}
+ */
+declare class FolderConfig extends FormApplication {}
+
+/**
+ * An Image Popout Application
+ * Provides optional support to edit the image path being viewed
+ * @type {Application}
+ * @params image {String}       The image being viewed
+ * @params options {Object}     Standard Application rendering options
+ * @params onUpdate {Function}  An optional callback function which should be triggered if the Image path is edited
+ */
+declare class ImagePopout extends FormApplication {}
+
+/**
+ * Template Measurement Config Sheet
+ *
+ * @param template			The template object being configured
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/foundry-vtt-api/types/apps/forms/forms.d.ts;
  Line: 27;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.